torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -108,7 +108,11 @@ std::pair<IValue, IValue> getFunctionTuple(
   static const std::string torch_prefix("__torch__");
   static const std::string class_prefix("__torch__.torch.classes");
 
-  for (const TypePtr& t : mobile_code.types_) {
+  for (const TypePtr& ty : mobile_code.types_) {
+    auto t = ty;
+    if (auto dyn = t->castRaw<c10::DynamicType>()) {
+      t = dyn->fallback();
```

Comment:
Is it always the case? Or does it only apply to some types?

Reply:
@cccclai this branch is only taken when we call save_for_mobile, and load it back, and re-save it, which happens in flatbuffer and lite interpreter unit tests.
For the one major flow we support (server jit module - save -> mobile module), this branch is never taken.

Reply:
In the future, if we migrate to flatbuffer completely, the if statement will always be true?